roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -32,7 +32,7 @@ discovery:
 
 gateway:
   expected_master_nodes: ${NODE_QUORUM}
```

Comment:
Shouldn't `expected_master_nodes` be the total cluster size?  If you keep this as a quorum value, then for a `>=` three node cluster, it will start recovering the state of the cluster when 2 nodes are present.  This means it will start to move/copy data to restore replicas and rebalance the cluster.  We only want this to happen when we exceed the `recover_after_time` value.